projects/core/src/auth/auth.module.ts

```diff
@@ -4,7 +4,7 @@ import { ClientAuthModule } from './client-auth/client-auth.module';
 import { UserAuthModule } from './user-auth/user-auth.module';
 
 @NgModule({
-  imports: [CommonModule, ClientAuthModule.forRoot(), UserAuthModule.forRoot()],
+  imports: [CommonModule, UserAuthModule.forRoot(), ClientAuthModule.forRoot()],
```

Comment:
Would be nice if we can add a comment here, stating that the order of imports matters, and link to an issue on GH which explains why it's important. What do you think?